tests/e2e/set-ffi-env-e2e

```diff
@@ -37,6 +37,19 @@ source "${SCRIPT_DIR}"/lib/diskutils
 source "${SCRIPT_DIR}"/lib/repoutils
 
 # GLOBALS
+# Configuration Constants
+readonly BLUECHI_CONTROLLER_CONF="/etc/bluechi/controller.conf"
+readonly BLUECHI_AGENT_CONF="/etc/bluechi/agent.conf.d/00-default.conf"
```

Comment:
Why use the conf.d directory for bluechi-agent, but not for the controller?

Reply:
Changed Thank you